### PR TITLE
Docs/create vuestic

### PR DESCRIPTION
**Pull request by m0ksem**
Docs for https://github.com/epicmaxco/vuestic-ui/pull/2828 (must be merged after #2828)

Just docs here for review. Find something unclear.

### CODE REVIEW

File: packages/create-vuestic/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+  const answers = await useUserAnswers()
+
+  console.log('')
+
+  await scaffoldProject(answers)
+  await addVuestic(answers)
+  await addAgGrid(answers)
+  await initGit()
+  await installDeps()
```
Comment (LighthouseKeeperYN) [inline packages/create-vuestic/src/index.ts:27]: Do they have to be sequential? If not you can use Promise.all to resolve them in parallel.

**Review by LighthouseKeeperYN — APPROVED** [review]
lgtm

File: packages/create-vuestic/src/index.ts (hunk 1 above)
Comment (m0ksem) [reply packages/create-vuestic/src/index.ts:27]: Most of the yes. We need to scaffold project before adding vuestic, also deps should be installed after all complete. For me, it is just clear to have it all sequential.

**Review by aluarius — APPROVED** [review]

File: packages/create-vuestic/src/steps/2.addVuestic/vue-app.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  ])
+  await writeFile(mainPath, mainSource)
+
+  // Add fonts
+
+  const htmlPath = resolvePath(process.cwd(), projectName, 'index.html')!
+  let htmlSource = await readFile(htmlPath, 'utf-8')
+  htmlSource = insertHead(htmlSource, [
+    `<link href="https://fonts.googleapis.com/css2?family=Source+Code+Pro&family=Source+Sans+Pro:wght@400;700&display=swap" rel="stylesheet">`,
+    `<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">`
```
Comment (rustem-nasyrov) [inline packages/create-vuestic/src/steps/2.addVuestic/vue-app.ts:43]: What if users don't have access to these CDNs?

File: packages/create-vuestic/src/utils/package-manager.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    return manager
+  }
+  // This environment variable is set by npm and yarn but pnpm seems less consistent
+  const userAgent = process.env.npm_config_user_agent
+
+  if (userAgent) {
+    if (userAgent.startsWith("yarn")) {
+      manager = "yarn"
+    } else if (userAgent.startsWith("pnpm")) {
+      manager = "pnpm"
```
Comment (rustem-nasyrov) [inline packages/create-vuestic/src/utils/package-manager.ts:15]: I saw that nuxt recommends to use `--shamefully-hoist` flag on installing. Maybe when is project is generating will be nice to add `.npmrc` file? Where we can put this flag.

**Review by rustem-nasyrov — COMMENTED** [review]
Great work!

File: packages/create-vuestic/src/steps/2.addVuestic/vue-app.ts (hunk 2 above)
Comment (m0ksem) [reply packages/create-vuestic/src/steps/2.addVuestic/vue-app.ts:43]: Then he will need to pay for internet. I can't see any possibility of not having access to CDN. 
Btw, it is possible to change CDN links and install fonts manually. But I would recommend to use CDN for fonts and icons.

File: packages/create-vuestic/src/utils/package-manager.ts (hunk 3 above)
Comment (m0ksem) [reply packages/create-vuestic/src/utils/package-manager.ts:15]: Nudt already does it, so we don't need to.

File: packages/create-vuestic/src/steps/2.addVuestic/vue-app.ts (hunk 2 above)
Comment (rustem-nasyrov) [reply packages/create-vuestic/src/steps/2.addVuestic/vue-app.ts]: @m0ksem https://github.com/epicmaxco/vuestic-ui/issues/2796

**Pull request merged**